spec/rails_helper.rb

```diff
@@ -75,3 +78,43 @@
     with.library :rails
   end
 end
+
+VCR.configure do |c|
+  # This is the directory where VCR will store its "cassettes", i.e. its
+  # recorded HTTP interactions.
+  c.cassette_library_dir = "spec/cassettes"
+
+  # This line makes it so VCR and WebMock know how to talk to each other.
+  c.hook_into :webmock
+
+  # This line makes VCR ignore requests to localhost. This is necessary
+  # even if WebMock's allow_localhost is set to true.
+  c.ignore_localhost = true
+
+  # ChromeDriver will make requests to chromedriver.storage.googleapis.com
+  # to (I believe) check for updates. These requests will just show up as
+  # noise in our cassettes unless we tell VCR to ignore these requests.
+  c.ignore_hosts "chromedriver.storage.googleapis.com"
+
+  c.preserve_exact_body_bytes do |http_message|
+    http_message.body.encoding.name == 'ASCII-8BIT' ||
+      !http_message.body.valid_encoding?
+  end
+
+  # Private environment variables must be set during test run to generate real data for the VCR
+  # They then are then hidden
```

Comment:
Here and bellow I'm adding support for obfuscating environment variables provided in this list (so they don't appear in the "VCR" generated)

Reply:
Basically hiding them from when the VCR takes a snap shot so sensitive info is not included right?

Reply:
yup!